glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt

```diff
@@ -192,6 +192,9 @@ class GleanTest {
             serverEndpoint = "http://" + server.hostName + ":" + server.port
         ))
 
+        // Send an "empty" metrics ping to reset the foreground_count
```

Comment:
Oof.

Reply:
Shouldn't `resetGlean` take care of that? It has `clearStores = true`

Reply:
Yeah -- I really struggled to make this test work, and thought this was required, but it seems in the end it isn't.  Removing.